spec.go

```diff
@@ -13,5 +13,6 @@ type Interface interface {
 	ExtClient() apiextensionsclient.Interface
 	G8sClient() versioned.Interface
 	K8sClient() kubernetes.Interface
-	RestConfig() *rest.Config
+	RESTClient() rest.Interface
+	RESTConfig() *rest.Config
```

Comment:
This is a breaking change unfortunately. `Rest` becomes `REST`. But I don't think it will hurt us a lot.

Reply:
Pawel I'm fine with the change but please do a #news-dev post. We just hit this in app-operator e2e.

Reply:
Sorry. I thought to do so but then I forgot.

Reply:
@kopiczko Any issue tickets for this? I hit this problem on `clsuter-autoscaler-app` repo.

Reply:
It's a pain but this is a problem that happens sometimes with vendoring breaking changes. There was a #news-dev post which is our usual way of notifying.

https://gigantic.slack.com/archives/C04TGHDEF/p1570108590032200

Reply:
Yeah not much I can do about it other than introducing backward compatible `RestConfig` calling `RESTConfig`. But I'd like to avoid it if possible.